### DISCUSSION TITLE
We are using enterprise version of github, license got expired recently. If we upgrade github from current version 3.7.6 to 3.8.1, does it have any issues because of expired license ?

### DISCUSSION THREAD

**Discussion by ashok60789** [community/community#72626] [Enterprise]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

We are using enterprise version of github, license got expired recently. If we upgrade github from current version 3.7.6 to 3.8.1, does it have any issues because of expired license ? 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
The [release notes](https://docs.github.com/en/enterprise-server@3.8/admin/release-notes) will share any known issues we're aware of, but we recommend you upgrade to the latest release of the 3.8 series.
